test/test_stats.py

```diff
@@ -0,0 +1,53 @@
+'''Test our stats utility'''
+
+import mock
+import os
+import unittest
+
+import simplejson as json
+
+from nsq.stats import Nsqlookupd
+from nsq.http import nsqd
+from common import IntegrationTest
+
+
+class TestStats(IntegrationTest):
+    '''Test our stats utility.'''
+
+    nsqd_ports = (14150, 14152)
```

Comment:
I agree in hindsight, it's not the most elegant interface. But, now's not the place to clean it up.